Electrify.Dlms/Server/GXDLMSBase.cs

```diff
@@ -76,11 +76,7 @@ public class GXDLMSBase : GXDLMSSecureServer
     static readonly object FileLock = new object();
     static string GetdataFile()
     {
-        // Get writeable directory
-        string dataDirectory = FileSystem.AppDataDirectory;
-    
-        // Define path to data.csv
-        return Path.Combine(dataDirectory, "data.csv");
+        return Path.Combine(Path.GetDirectoryName(typeof(GXDLMSBase).Assembly.Location), "data.csv");
```

Comment:
was the old code here fixing something before? i think josh or james looked at it or something like this?